### PR TITLE
Add `PickTag()` for `ctags` support

### PR DESCRIPTION
**Pull request by teoljungberg**
- Pass all present `tags` files to `pick`

### CODE REVIEW

File: plugin/pick.vim [hunk 1]
```diff
@@ -68,3 +72,16 @@ function! s:BufferListCommand()
...
+  let l:tag_file = ""
+
+  for tfile in split(&tags, ",")
+    if filereadable(expand(tfile)) == 1
+      let l:tag_file = tfile
+      break
+    endif
+  endfor
+
+  return "awk -F$'\t' '{print $1}' " . l:tag_file . "  | sort -u | grep -v '^!'"
```
Comment (calleluks) [inline plugin/pick.vim]: Could we support multiple tag files by replacing unescaped commas (not preceded by a backslash) with spaces and passing the resulting string directly to awk?

Comment (teoljungberg) [inline plugin/pick.vim]: Good idea, `awk` takes it's arguments separated by spaces. So we can select the files that are readable from `&tags` and then join them by spaces when we pass the list to `awk`

Done in e798eaf

File: plugin/pick.vim [hunk 2]
```diff
@@ -68,3 +72,15 @@ function! s:BufferListCommand()
   let buffers = map(bufnrs, 'bufname(v:val)')
   return 'echo "' . join(buffers, "\n") . '"'
 endfunction
+
+function! s:TagCommand()
+  let l:tag_files = []
+
+  for tfile in split(&tags, ",")
+    if filereadable(expand(tfile)) == 1
```
Comment (calleluks) [inline plugin/pick.vim]: Testing file readability is susceptible to race conditions.

Could this be avoided by using cat to to join the existing files before invoking awk?

``` viml
function! s:TagCommand()
  return "cat " . join(split(&tags, ","), " ") . " 2> /dev/null | awk -F$'\t' '{print $1}' | sort -u | grep -v '^!'"
endfunction
```

Comment (teoljungberg) [inline plugin/pick.vim]: I wonder if race conditions will actually be a concern in a vim plugin.

But interesting idea, I like it alot! Fixed in 0b1472d

**Comment by calleluks** [comment]
Looks good to squash and merge!

**Comment by teoljungberg** [comment]
Thanks for reviewing @calleerlandsson

**Pull request merged**